policy/lib/bundles.rego

```diff
@@ -15,83 +18,165 @@ disallowed_task_reference(tasks) := {task |
 # Returns a subset of tasks that use an empty bundle reference.
 empty_task_bundle_reference(tasks) := {task |
 	some task in tasks
-	bundle(task) == ""
+	refs.task_ref(task).bundle == ""
 }
 
 # Returns a subset of tasks that use bundle references not pinned to a digest.
 unpinned_task_bundle(tasks) := {task |
 	some task in tasks
-	ref := image.parse(bundle(task))
+	ref := image.parse(_bundle_ref(task, data["task-bundles"]))
```

Comment:
This function probably doesn't need to use `_bundle_ref` since it only cares about the digests.